core/src/stored/backends/droplet_device.cc

```diff
@@ -361,14 +361,26 @@ bool droplet_device::CheckRemote()
     if (!initialize()) { return false; }
   }
 
-  if (check_path("/") != DPL_SUCCESS) {
-    Dmsg1(100, "CheckRemote(%s): failed\n", prt_name);
-    return false;
-  }
+  auto status = check_path("/");
 
-  Dmsg1(100, "CheckRemote(%s): ok\n", prt_name);
+  const char* h = dpl_addrlist_get(ctx_->addrlist);
+  std::string hostaddr{h != nullptr ? h : "???"};
 
-  return true;
+  switch (status) {
+    case DPL_SUCCESS:
+      Dmsg1(100, "Host is accessible: %s\n", hostaddr.c_str());
+      return true;
+    case DPL_ENOENT:
+      Dmsg2(100,
+            "Host is accessible: %s (%s), but should be configured to accept "
```

Comment:
Is this the only reason that causes this return value? If not, then the error message should say something like "probably the host should be configured...."